public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -2,7 +2,7 @@
 
 block includes
   include ../_util-fns
-  - var _appRoutingTsVsAppComp = 'app.routing.ts'
+  - var _appRoutingTsVsAppComp = 'app.module.ts'
```

Comment:
Local Jade variable names used to render TS and Dart prose from the same source. Such variable names do not appear in the rendered HTML.